### ISSUE TITLE
Ivy not compatible

### ISSUE THREAD

**Issue by raphael22** [lentschi/ngx-ellipsis#75]
Hi

Is your library could be publish as Ivy for Angular 13+ ?

```
Processing legacy "View Engine" libraries:
- ngx-ellipsis [es2015/esm2015] (https://github.com/lentschi/ngx-ellipsis)
- ngx-ellipsis [main/umd] (https://github.com/lentschi/ngx-ellipsis)
Encourage the library authors to publish an Ivy distribution.
```

**Comment by lentschi** [comment]
Hi @raphael22 !

To be honest I'm not sure if that is possible without losing support for angular < 12 at the same time.
If anyone has any ideas on how to do that, please create a PR!

Eventually of course, I _will_ upgrade and move the requirements to angular >=12, but I think it's a bit early for that. (The supported versions would just be `>=12.0.0 <14.0.0`, which I find quite narrow. I Guess as soon as angular 14 is official, I'll do it.)

**Comment by raphael22** [comment]
Yup, since Angular 13 drop view-engine I don't think you can make it compatible for app with enableIvy: false.

**Comment by capraibex** [comment]
> Eventually of course, I _will_ upgrade and move the requirements to angular >=12, but I think it's a bit early for that. (The supported versions would just be `>=12.0.0 <14.0.0`, which I find quite narrow. I Guess as soon as angular 14 is official, I'll do it.)

Since June 2nd, Angular 14 is now official. Are there any plans to move the requirements to >= 12 or at least update the peerDependencies to `<15.0.0` ?

**Comment by lentschi** [comment]
@capraibex Thanks for the reminder! `ngx-ellipsis@4.0.0` is published and supports angular `>=12.0.0 <15.0.0`.
